lib/adapters/websql/utils.js

```diff
@@ -147,8 +147,9 @@ function compactRevs(revs, docId, tx) {
   });
 }
 
-function unknownError(callback) {
+function websqlError(callback) {
   return function (event) {
+    console.error('WebSQL threw an error', event);
```

Comment:
These kinds of things can be really tricky to debug, so I think it's best to just log an error so the user can see it immediately. We do a similar thing in idb as well.